spec/system/admin/reports/sales_tax/sales_tax_totals_by_order_spec.rb

```diff
@@ -104,9 +104,7 @@
     end
 
     it "generates the report" do
-      login_as admin
-      visit admin_reports_path
-      click_on "Sales Tax Totals By Order"
+      visit_sales_tax_totals_by_order
```

Comment:
Good idea but ideally we don't go the whole path in every spec. It's good to verify once that this path works but then we can navigate straight to the report page without going to the overview first. That saves some time.